src/jit/compiler.h

```diff
@@ -3114,12 +3114,16 @@ class Compiler
     bool compSupportsHWIntrinsic(InstructionSet isa);
     bool isScalarISA(InstructionSet isa);
     static int ivalOfHWIntrinsic(NamedIntrinsic intrinsic);
+    unsigned simdSizeOfHWIntrinsic(NamedIntrinsic intrinsic, CORINFO_SIG_INFO* sig);
```

Comment:
Can this one not be static anymore?

Reply:
No, it is calling `getSIMDTypeSizeInBytes`.